src/Starcounter.Linq/QueryProvider.cs

```diff
@@ -19,6 +19,6 @@ public QueryProvider(IQueryContext queryContext)
 
         object IQueryProvider.Execute(Expression expression) => _queryContext.Execute(expression);
 
-        T IQueryProvider.Execute<T>(Expression expression) => (T) _queryContext.Execute<T>(expression);
```

Comment:
Please remove the `_` usage, by our own guides we do not recommend using `_` in C#.

Reply:
Fixed